modules/exploits/unix/dhcp/rhel_dhcp_client_command_injection.rb

```diff
@@ -0,0 +1,76 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
`require 'msf/core'` shouldn't be necessary

Reply:
Removed. This should fix the test